legacy/account/account_s3.ftl

```diff
@@ -268,17 +268,7 @@
                     [
                         "function sync_code_bucket() {"
                     ] +
-                        scriptSyncContent +
-                    [
-                        "}",
-                        "#",
-                        "case $\{STACK_OPERATION} in",
-                        "  create|update)",
-                        "    sync_code_bucket || return $?",
-                        "    initialise_registries ${s3RegistryIds} || return $?",
-                        "    ;;",
-                        " esac"
-                    ]
+                        scriptSyncContent
```

Comment:
It's been a while so I may be reading this wrong, but it looks like `scriptSyncContent` will need a `}` somewhere in there? 

> `scriptSyncContent + [ "}" ]`